src/Table.php

```diff
@@ -594,12 +594,12 @@ public function renderRow()
      *
      * @return Jquery
      */
-    public function onRowClick($action)
+    public function onRowClick($action, array $excludeSelector = [])
     {
         $this->addClass('selectable');
         $this->js(true)->find('tbody')->css('cursor', 'pointer');
 
-        return $this->on('click', 'tbody>tr', $action);
+        return $this->on('click', 'tbody tr td' . ($excludeSelector ? ':not(' . implode(',', $excludeSelector) . ')' : ''), $action);
```

Comment:
why not simply exclude a checkbox control?

Reply:
More flexible this way. You may decide to exclude any table cell you need.

Reply:
adding `Checkbox::CHECKBOX_COLUMN_CSS` constant seems not needed for me, but if we absolutely need to use exclusing selector, I am ok with it